clients/go.mod

```diff
@@ -9,4 +9,17 @@ require (
 	github.com/stretchr/testify v1.7.0
 )
 
+require (
```

Comment:
Is this auto-generated? Why does it exist separately from the require chunk above?

Reply:
Yes this is auto-generated. Since Go 1.17, the module graph enables pruning and lazy loading. The second `require` block contains indirect dependencies.